homeworks/09-node-chokidar/package.json

```diff
@@ -7,13 +7,21 @@
   "scripts": {
     "build": "tsc",
     "start": "node ./bin/app.js",
```

Comment:
This script should be combined:
```
    "_build" : "tsc",
    "_start-app": "node ./bin/app.js",
    "start": "yarn _build && yarn _start-app",
```

<img width="481" alt="Screenshot 2021-05-31 at 11 59 26" src="https://user-images.githubusercontent.com/364880/120168547-a85a9380-c207-11eb-99a4-5f8a2461c104.png">

Otherwise running `yarn start` throws error of missing file in `bin` directory.